### ISSUE TITLE
WHY YOU DONT FIXED IT??!?!?!?!?! VALVE=SCAM!!!!

### ISSUE THREAD

**Issue by Magvay89** [ValveSoftware/Dota2-Gameplay#5517]
### Item name

Weekly Quest Chest

### Description

https://github.com/ValveSoftware/Dota2-Gameplay/issues/4941

This problem is still not fixed!!!
What the hell?!
Bonus week, all chests again give 1000 points.
This problem since September, you know about it, and do nothing!!! You just don't care about your users.

### Example Match ID and Timestamp

No match. Problem in battlepass.

**Comment by gaia1991** [comment]
very true

**Comment by tomsawyerr** [comment]
YES ALL THEY WANT IS MONEY, THEY DONT GIVE A FUCK ABOUT FIXING SHIT OR FIXING FUCKING GRIEFERS IN EVERY FUCKING GAME. IF I KNEW THESE FUCKERS WOULD BE SO FUCKING COCKSUCKERS, I WOULDNT HAVE BOUGHT SO MANY FUCKING LEVELS OF FUCKING BATTLE PASS. COCKSUCKERS.
